src/components/Display.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+
+function Display(props) {
+  const { result } = props;
+  return (
+    <div>{result}</div>
+  );
+}
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
```

Comment:
Kindly add/set the default value of the result props to `0`

Reply:
I have set the default value of the result props to `0` and also implemented the changes suggested